vumi/transports/vumi_bridge/tests/test_vumi_bridge.py

```diff
@@ -183,17 +182,15 @@ def test_reconnecting(self):
         self.assertFalse(transport.reconnect_call)
 
 
-class TestGoConversationServerTransport(TestGoConversationTransportBase):
+class TestGoConversationTransport(TestGoConversationTransportBase):
 
-    transport_class = GoConversationServerTransport
+    transport_class = GoConversationTransport
 
     def test_server_settings_without_configs(self):
         return self.assertFailure(self.get_transport(), ConfigError)
 
     def get_configured_transport(self):
-        return self.get_transport(
-            message_path='messages.json', event_path='events.json',
-            web_port='0')
+        return self.get_transport(web_path='', web_port='0')
```

Comment:
We should probably set `web_path` to something that is not `''` to demonstrate that it's working.